### DISCUSSION TITLE
Prevent syncing on boot

### DISCUSSION THREAD

**Discussion by Imaginous** [marcone/teslausb#845] [Q&A]
Is it possible to prevent syncing on boot?

I have a fairly large music library which takes a few minutes to scan. During this time USB is not mounted to the car and music is not available.

My idea is to mount music (and cam) directly and only trigger a sync when reconnecting to WiFi.
This way it will sync when I return home but USB is directly available when I leave home.

If I want to sync my music before my drive, I can trigger a manual sync.

If it's not possible yet, then this might be a feature ;)

**Comment by marcone** [comment]
If your music library is so large that it takes several minutes to scan, then I'd say you're better off using a separate USB drive for it.

**Reply by Imaginous** [reply]
But since the USB port in the Model 3 Highland is located very inconvenient. Your tool makes adding music so easy.

But I understand your point of view.

**Reply by marcone** [reply]
You could consider disabling music syncing and copy music using the web interface instead.
